CONTRIBUTING.rst

```diff
@@ -93,30 +86,25 @@ and for a new **feature**::
 
   $  git pull-request  -m  '<short_change_message> #<issue number>' -b next
 
-**Step 5**: Watch the pull request for comments and reviews. For any pull requests update,
-please try to squash/amend your commits to avoid "in-between" commits.
+**Step 5**: Watch the pull request for comments and reviews. For any pull requests update, please try to squash/amend your commits to avoid "in-between" commits.
 
 Automatic Review
 ----------------
 
-Every submitted pull request will automatically be run through automated review and
-testing(nosetests) with Travis.
+Every submitted pull request will automatically be run through automated review and testing with Travis.
 
 Human Review
 ------------
 
 Anyone is welcome to review merge requests and make comments!
 
-All collaborators, thus the Rucio core development team can approve, request
-changes or close pull requests. Merging of approved pull requests is done by the Rucio
-development lead.
+All collaborators, thus the Rucio core development team can approve, request changes or close pull requests. Merging of approved pull requests is done by the Rucio development lead.
```

Comment:
I think we can remove the `All collaborators` part. It's somewhat redundant to Rucio development team. (Maybe ditch the core too)